onnx2tf/onnx2tf.py

```diff
@@ -738,23 +739,21 @@ def convert(
                                 MEAN,
                                 STD,
                             ]
-            else:
-                if output_integer_quantized_tflite \
-                    and quant_calib_input_op_name_np_data_path is not None:
-                    for param in quant_calib_input_op_name_np_data_path:
-                        input_op_name = str(param[0])
-                        numpy_file_path = str(param[1])
-                        calib_data = np.load(numpy_file_path)
-                        if data_count == 0:
-                            data_count = calib_data.shape[0]
-                        mean = param[2]
-                        std = param[3]
-                        calib_data_dict[input_op_name] = \
-                            [
-                                calib_data.copy(),
-                                mean,
-                                std,
-                            ]
+            elif quant_calib_input_op_name_np_data_path is not None:
```

Comment:
Reject this modification because it is necessary to generate a Float32 model, as per the comment in Line.670.